docs/config_search.md

```diff
@@ -150,7 +151,19 @@ as the range for the `max_batch_size` to `[1]`, it will no longer be a valid
 Triton Model Configuration.
 
 The configuration sweep described above, will sweep through 8 configs = (2
-`max_batch_size`) * (2 `max_queue_delay_microseconds`) * (2 `instance_group`) values.
+`max_batch_size`) _ (2 `max_queue_delay_microseconds`) _ (2 `instance_group`) values.
```

Comment:
underscores now?

Reply:
This is the result of YAPF formatting. It looks identical when viewing the document

Reply:
I think you replied to the wrong comment, and/or did a bad search and replace. These should not be underscores.

![image](https://user-images.githubusercontent.com/50968584/188967704-943af327-0c75-4594-a0d6-0c94ca4e2b7d.png)

Reply:
You are correct. This formatting did need to be fixed. YAPF converted it into underscores, but `/*` preserves them.